packages/stylelint-config-airbnb/.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
+dist: trusty
+language: node_js
+cache:
+  directories:
+    - node_modules
+node_js:
+  - '6'
```

Comment:
Done in https://github.com/airbnb/css/pull/23/commits/ea3ef3d0edda84a638633ee079941a63d4b192b4